src/utils/tickets.js

```diff
@@ -2,16 +2,62 @@ import ScreenshotManager from './screenshots.js';
 import ResourceManager from '../manager/ResourceManager.js';
 import getBrowser from '../utils/browser.js';
 import ArtifactManager from '../manager/ArtifactManager.js';
+import { config } from '../index.js';
+
+import fetch from 'node-fetch';
+
+const getSafeBrowsingIndex = async URL => {
+  console.log(`Getting Google Safe Browsing information...`);
+  const safeBrowsingURL =
+    'https://safebrowsing.googleapis.com/v4/threatMatches:find?key=' +
+    config.google_safe_browsing_api_key;
+  const request = {
+    client: {
+      clientId: '4950',
+      clientVersion: '0.0.1',
+    },
+    threatInfo: {
+      threatTypes: ['MALWARE'],
+      platformTypes: ['ANY_PLATFORM'],
+      threatEntryTypes: ['URL'],
+      threatEntries: [{ url: URL }],
+    },
+  };
+
+  try {
+    const res = await fetch(safeBrowsingURL, {
+      method: 'POST',
+      body: JSON.stringify(request),
+      responseType: 'application/json',
+    });
+    const json = await res.json();
+    if (json.matches === undefined) {
+      console.log('No malware found');
+      return null;
+    } else {
+      const match = json.matches[0];
```

Comment:
Curious about taking the first element here - what else is in that array?

Reply:
The API returns an array, because we can test multiple URLs in 1 request and there might be multiple matches.

Reply:
Shouldn't we also return the full array, then? We can loop through and show all results on the frontend.